README.md

```diff
@@ -21,18 +21,30 @@ You need:
 1. [Datadog API key](https://app.datadoghq.com/organization-settings/api-keys)
 2. and a [Datadog Application Key](https://app.datadoghq.com/organization-settings/application-keys)
 
-####  Configuration
+#### Add Rake Tasks
 
-If using in a Rails project, setup an initializer to configure the gem:
+Add the following to your Rakefile:
 
 ```ruby
-Datadog::ServiceCatalog.configure do |config|
-  config.datadog_api_key = '...'
-  config.datadog_application_key = '...'
-  config.markdown_file = 'service_catalog.md'
+require 'datadog/service_catalog/rake_tasks'
+
+namespace :service_catalog do
+  Datadog::ServiceCatalog.configure do |config|
+    config.datadog_api_key = '..'
```

Comment:
Don't we already have a data dog api key in the .env files? If yes, could we use that?

Reply:
We do not; it needs to be added to the .envs. I am putting together an Ansible PR do that. When you add to the various services, you will need to add them to CircleCi env